pom.xml

```diff
@@ -24,13 +24,14 @@
         <kotlin.version>1.8.21</kotlin.version>
         <felles.version>1.20221202080911_bcf8f33</felles.version>
         <common-java-modules.version>2.2021.09.17_07.09-67428a6422cc</common-java-modules.version>
-        <kontrakter.version>2.0_20230313155154_f9438bd</kontrakter.version>
+        <kontrakter.version>3.0_20230523132455_9170af0</kontrakter.version>
```

Comment:
Versjon 3 er basert på jakarta, og vi får ikke oppgradert tilbake ennå.
Det er mulig det skaper en del krøll å ha avhengigheter til både javax og jakarta, så anbefaler å ikke gjøre dette.

2 alternativ:
* Ikke oppdatere kontrakter, men mappe til tema selv her
* Branche ut kontrakter ut fra en tidligere versjon og få inn endringene for Tema